internal/sgdisk/sgdisk.go

```diff
@@ -121,6 +121,11 @@ func (op *Operation) Commit() error {
 	if _, err := op.logger.LogCmd(cmd, "deleting %d partitions and creating %d partitions on %q", len(op.deletions), len(op.parts), op.dev); err != nil {
 		return fmt.Errorf("create partitions failed: %v", err)
 	}
+	// In contrast to similar tools, sgdisk does not trigger the update of the kernel partition table
+	cmd = exec.Command(distro.PartprobeCmd(), op.dev)
+	if _, err := op.logger.LogCmd(cmd, "re-reading of %d deleted partitions and %d created partitions on %q", len(op.deletions), len(op.parts), op.dev); err != nil {
```

Comment:
Hmm, that log message doesn't seem very clear. We already logged the number of deleted and created partitions above, so let's just go for

```suggestion
	if _, err := op.logger.LogCmd(cmd, "triggering partition table reread on %q", op.dev); err != nil {
```

?